Introductory Tests/EquationsOfMotion.py

```diff
@@ -1,9 +1,13 @@
 # General
 import numpy as np
+from math import pi, radians
 
 # For computing acceleration and potential
 import polyhedral_gravity as model
 
+# For computing rotations of orbits
+from pyquaternion import Quaternion
```

Comment:
has to be added to environment.yml I think

Reply:
also, you may want to fix the numpy version, I had to install numpy==1.23.5 to avoid getting weird errors (problems with 1.24 I think)

Reply:
![image](https://user-images.githubusercontent.com/2656685/222436018-581460c6-4fbe-4216-95e4-51dcb369be19.png)